build.gradle

```diff
@@ -59,6 +60,7 @@ dependencies {
 	api 'org.jsoup:jsoup:1.17.2'
 	implementation 'org.json:json:20231013'
 	implementation 'androidx.collection:collection:1.4.0'
+	implementation "org.jetbrains.kotlinx:kotlinx-serialization-json:1.6.3"
```

Comment:
Is it possible to do this without adding new dependencies? Using org.json?

Reply:
fixed.